frontend/src/APIClients/UserAPIClient.ts

```diff
@@ -60,4 +60,20 @@ const updateUserById = async (
   }
 };
 
-export default { getUserById, updateUserById };
+const deleteUserById = async (id: string): Promise<boolean> => {
+  const bearerToken = `Bearer ${getLocalStorageObjProperty(
+    AUTHENTICATED_USER_KEY,
+    "accessToken",
+  )}`;
+
+  try {
+    await baseAPIClient.delete(`/users?userId=${id}`, {
+      headers: { Authorization: bearerToken },
+    });
+    return true;
+  } catch (error) {
+    return false;
+  }
+};
+
```

Comment:
this function will be overwritten

Reply:
this will remain the same, just the backend function corresponding to the route that this function calls is being changed accordingly. A pr is open for that (https://github.com/uwblueprint/community-fridge-kw/pull/184). We can do the end to end testing of delete in the pr that gets merged in after. It will make testing a bit easier in the other pr if we merge this pr first (cuz dont have to delete users using postman)